push-image/docker.js

```diff
@@ -0,0 +1,29 @@
+const core = require('@actions/core');
+
+const { execFileSync } = require('child_process');
+
+function docker(command, ...args) {
```

Comment:
This function basically replaces [docker-cli-js](https://www.npmjs.com/package/docker-cli-js)